css/style.css

```diff
@@ -0,0 +1,524 @@
+    width: 90%;
+    height: 20px;   
+    position: fixed;
+    top: 0px;
+    left: 5%;
+    z-index: 1;   
+}
+
+.container {
+    width: 90%;
+    margin: auto;
+    margin-top: 20px;
+    overflow: hidden;
+    background-color: white;
+    z-index: 0;
+}
+
+.container > header {
+    position: relative;
+}
+
+a.logo-link:empty {
+    background-image: url('http://i67.tinypic.com/14ugk5j.png');
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 200px;
+    height: 50px;
+    display: inline-block;  
+    position: absolute;
+    top: 2px;
+    right: 25px;
+}
+
+.container > header > nav {
+    display: inline-flex;
+    align-items: center;
+    z-index: 1;
+    overflow: hidden;
+    width: 100%;
+    height: 45px;
+}
+
+.container > header > nav >  a.main-link {
+    width: 12.5%;
+    text-decoration: none;
+    justify-content: space-between;
+    color: silver;
+    font-size: 20px;
+    padding: 5px;
+    text-align: center;
+    font-weight: bold;
+}
+
+.container > header > nav >  a.main-link:hover {
+    border: 5px double grey;
+    transition-duration: 250ms;
+    transition-property: all;
+}
+
+.container > header > nav > a.main-link:active {
+    border: 5px solid grey;
+    font-weight: normal;
+}
+
+.container > header > div.home-header {
+    display: flex;
+    justify-content: center;
+    margin: 150px 0 100px;
+}
+
+.get-started-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+
+.get-started-container > div {
+    margin: 20px 25px;
+}
+
+.slogan {
+    font-size: 75px;
+    text-align: center;
+}
+
+.get-started-button {
+    margin-top: 30px;
+    cursor: pointer;
+}
+
+.get-started-button > span {
+    font-family: "Fira Mono";
+    border: 1px solid black;
+    padding: 5px 25px;
+}
+
+.get-started-button:hover > span {
+    border: 1px solid grey;
+    background-color: grey;
+    color: white;
+    transition-duration: 500ms;
+    transition-property: all;
+}
+.get-started-button:active > span {
+    background-color: silver;
+    border: 1px dotted white;
+    transition-duration: 125ms;
+    transition-property: all;
+}
+    
+.get-started-image:empty {
+    background-image: url('http://i68.tinypic.com/f6lqu.png');
+    background-size: contain;
+    background-repeat: no-repeat;
+    width: 380px;
+    height:  380px;
+    border-radius: 100%;   
+}
+
+header {
+    border-bottom: 1px solid black;
+    margin: 35px auto;
+}
+
+.grouping-container {
+    display: flex;
+    justify-content: space-between;
+    margin: 15px auto;
+}
+
+.grouping-container > section {
+    margin: 0 20px;
+}
+
+.middle-img {
+    width: 100%;
+    height: auto;
+    margin: 25px 0;
+}
+
+footer {
+    margin-top: 30px;
+    border-top: 1px solid black;
+    padding-top: 30px;
+}
+
+footer > address {
+    line-height: 1.15;
+}
+
+footer > div.copyright-section {
+    margin: 30px auto;
+    text-align: center;
+}
+
+.header-image {
```

Comment:
What is this the start of? I had to scroll down to see this is a services page. A comment would be good in this case.